docs/pages/7_0_0_release_notes.md

```diff
@@ -17,7 +17,55 @@ This is a {{ site.pmd.release_type }} release.
 
 {% tocmaker is_release_notes_processor %}
 
-### New and noteworthy
+### 🚀 New and noteworthy
+
+#### Release Candidates
+
+PMD 7.0.0 is finally almost ready. In order to ease the upgrade from PMD 6 to 7, we are going to ship a couple of
+release candidates. These are official pre-release versions available on GitHub and Maven Central and can be used
+as usual (e.g. as a dependency).
+
+Using the release candidate is also a good opportunity to provide feedback via our
+[issue tracker](https://github.com/pmd/pmd/issues/new/choose).
+
+#### New official logo
+
+Many of you probably have already seen the new logo, but now it's time to actually ship it. The new logo
+was long ago decided (see [#1663](https://github.com/pmd/pmd/issues/1663)).
+We decided it's time to have a modernized logo and get rid of the gun. This allows to include
+the logo anywhere without offense.
+
+
+![New PMD Logo](images/logo/pmd-logo-300px.png)
```

Comment:
We have a couple of relative links which refer to files/pages inside the repo and the generated page. These won't work on e.g. Github Releases.

We probably should replace them with absolute links (https://pmd.github.io/latest/...) - which then only are valid after the release of pmd-7.0.0-rc1.

Reply:
I've introduced a page variable:

```
baseurl: https://docs.pmd-code.org/pmd-doc-7.0.0-SNAPSHOT/
```

This variable needs to be adjusted.

The ruletags generate a slightly different url:

```
url_prefix = "https://pmd.github.io/pmd-#{context["site.pmd.version"]}/"
```

Maybe that should be aligned? The ruletags url will only work when the release is done and the doc is uploaded to pmd/pmd.github.io.

Reply:
I implemented now a better solution that makes it work for the release notes rendering as well as the site rendering.

For site-rendering, we use relative urls, for release notes, we use the deployed url - similar what rule tags does.